src/feature/Form/Login/Login.tsx

```diff
@@ -0,0 +1,69 @@
+import { Link } from "react-router-dom";
+import { Button } from "components/Button/Button";
+import { Input } from "components/Input/Input";
+import { Label } from "components/Label/Label";
+import { InputError } from "components/Input/InputError/InputError";
+import { Welcome } from "../components/Welcome/Welcome";
+import { regexEmail } from "constants/regexEmail";
+import styles from "./Login.module.scss";
+import { useFormLogin } from "./hooks/useFormLogin";
+
+export const Login = () => {
+	const { handleSubmit, onSubmit, register, errors, isPending, error } =
+		useFormLogin();
```

Comment:
Wrzucanie obsługi formularza, którą otrzymujemy z react hook form do hooka, który robi inną logikę to robienie niepotrzebnego spaghetti. Ktoś kto by nad tym później pracował musiałby się przeklikać przez kilka modułów żeby wykonać zmiany w kodzie.

Ten custom-hook pozostawiłbym w roli komunikatora api + do obsługi statusu tej komunikacji.